### PR TITLE
Epic / Service PoC

### PR DESCRIPTION
**Pull request by SeferMirza**
Import all features developed for PoC purposes directly into blueprints-service
project without any documentation or testing.

- [x] Layers
  - [x] Data Access & Monitoring #21
  - [x] Http Server #20
  - [x] Rest Api & Documentation #22
  - [x] Configuration #23
  - [x] Testing #25
- [x] Features w/o Port
  - [x] Greeting/Swagger #24
  - [x] Logging/Request #24
  - [x] MockOverrider/FirstInterface
    - #24
    - #26
  - [x] Business #24
- [x] Service Port #27
- [x] Features w/ Port
  - [x] Core #18
  - [x] Exception Handling #29
  - [x] Database/* #28
  - [x] Orm/NHibernate #28

### CODE REVIEW

File: src/blueprints/Do.Blueprints.Service/Greeting/WelcomePage/WelcomePageGreetingFeature.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+using Do.Architecture;
+using Microsoft.AspNetCore.Builder;
+
+namespace Do.Greeting.WelcomePage;
+
+public class WelcomePageGreetingFeature : IFeature
+{
+    string _path;
```
Comment (cihandeniz) [inline src/blueprints/Do.Blueprints.Service/Greeting/WelcomePage/WelcomePageGreetingFeature.cs:8]: should be readonly

File: src/blueprints/Do.Blueprints.Service/Logging/LoggingExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿using Do.Architecture;
+using Do.Logging;
+
+namespace Do;
+
+public static class LoggingExtensions
+{
+    public static void AddLogging(this IList<IFeature> source, Func<LoggingConfigurator, IFeature> configure) => source.Add(configure(new()));
```
Comment (cihandeniz) [inline src/blueprints/Do.Blueprints.Service/Logging/LoggingExtensions.cs:8]: `IList` -> `List`

File: src/blueprints/Do.Blueprints.Service/MockOverrider/FirstInterface/MockOverriderMockFactory.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿using Do.Testing;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Do.MockOverrider.FirstInterface;
+
+public class MockOverriderMockFactory : DefaultMockFactory
+{
+    public override object Create(IServiceProvider serviceProvider, MockDescriptor mockRegistration)
```
Comment (cihandeniz) [inline src/blueprints/Do.Blueprints.Service/MockOverrider/FirstInterface/MockOverriderMockFactory.cs:8]: rename as `mockDescriptor`

**Review by cihandeniz — COMMENTED** [review]
some things to do for final touches

**Review by cihandeniz — APPROVED** [review]

**Review by dncsvr — APPROVED** [review]

**Pull request merged**